test/hexpm/accounts/audit_log_test.exs

```diff
@@ -42,6 +42,26 @@ defmodule Hexpm.Accounts.AuditLogTest do
       assert audit.params.user.username == user.username
       assert audit.params.user.handles.github == user.handles.github
     end
+
+    test "action billing.create", %{user: user} do
+      audit =
+        AuditLog.build(user, "user_agent", "billing.create", %{
+          "email" => "test@example.com",
```

Comment:
Is this the correct `params` structure we want to save for `billing.create`?

I found these params from `OrganizationController.update_billing/4`: https://github.com/hexpm/hexpm/pull/837/files#diff-cefbc7f8ca41c8340829f1e42dd11cdbL294-L298

Reply:
Yes, looks good, but see my comment above about including the organization.